From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: John Doe <john.doe@somewhere.on.planet>
Date: Fri, 9 May 2025 08:41:33 +0000
Subject: Patching kernel rockchip64 files sound/soc/codecs/Kconfig

Signed-off-by: John Doe <john.doe@somewhere.on.planet>
---
 sound/soc/codecs/Kconfig | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/sound/soc/codecs/Kconfig b/sound/soc/codecs/Kconfig
index 6a72561c4..27589e27e 100644
--- a/sound/soc/codecs/Kconfig
+++ b/sound/soc/codecs/Kconfig
@@ -49,10 +49,11 @@ config SND_SOC_ALL_CODECS
 	imply SND_SOC_AK4641
 	imply SND_SOC_AK4642
 	imply SND_SOC_AK4671
 	imply SND_SOC_AK5386
 	imply SND_SOC_AK5558
+	imply SND_SOC_AK7755
 	imply SND_SOC_ALC5623
 	imply SND_SOC_ALC5632
 	imply SND_SOC_AUDIO_IIO_AUX
 	imply SND_SOC_AW8738
 	imply SND_SOC_AW87390
@@ -624,10 +625,15 @@ config SND_SOC_AK5386
 
 config SND_SOC_AK5558
 	tristate "AKM AK5558 CODEC"
 	depends on I2C
 	select REGMAP_I2C
+	
+config SND_SOC_AK7755
+	tristate "AKM AK7755 CODEC"
+	depends on I2C
+	select REGMAP_I2C
 
 config SND_SOC_ALC5623
 	tristate "Realtek ALC5623 CODEC"
 	depends on I2C
 
-- 
Created with Armbian build tools https://github.com/armbian/build

